### PR TITLE
skipper-ingress: allow disabling filters

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -82,9 +82,8 @@ skipper_suppress_route_update_logs: "true"
 skipper_validate_query: "true"
 skipper_validate_query_log: "false"
 
-# skipper default filters
 skipper_default_filters: 'enableAccessLog(4,5) -> fifo(2000,20000,"3s")'
-# inline routes
+skipper_disabled_filters: ""
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:86]: Named similar to `skipper_default_filters`, i.e. did not use `skipper_ingress_` prefix

File: cluster/manifests/skipper/deployment.yaml [hunk 2]
```diff
@@ -201,6 +201,7 @@ spec:
           - "-read-timeout-server={{ .ConfigItems.skipper_read_timeout_server }}"
           - "-write-timeout-server={{ .ConfigItems.skipper_write_timeout_server }}"
           - '-default-filters-prepend={{ .ConfigItems.skipper_default_filters }}'
+          - "-disabled-filters={{ .ConfigItems.skipper_disabled_filters }}"
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml:204]: Empty is the default flag value so there is no need for if-empty check

**Comment by szuecs** [comment]
:+1:

**Comment by MustafaSaber** [comment]
👍  - I think my vote still doesn't count here

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**